cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -25,7 +24,7 @@ def pytest_generate_tests(metafunc):
 def testing_instance(request, setup_provider, provider):
     """ Fixture to provision instance on the provider
     """
-    instance = Instance.factory(random_vm_name('pwr-ctrl'), provider)
+    instance = Instance.factory(random_vm_name('pwr-c'), provider)
```

Comment:
oh i see, i feel that sideeffect is unintentional and should be canged in future

@psav do we want to do something about that (my personal preference would be to have more meaningful prefixes at the cost of randomization)